common/spacing/spacing-en.html

```diff
@@ -0,0 +1,70 @@
+---
+{
+	"title": "Spacing",
+	"language": "en",
+	"altLangPage": "spacing-fr.html",
+	"breadcrumbs":
+	[
+		{
+			"title": "GCWeb home",
+			"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+		}
+	],
+	"secondlevel": false,
+	"dateModified": "2023-07-04",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.p-0</code></dt>
+	<dd>Set padding to 0 for the element</dd>
+	<dt><code>.pl-2</code></dt>
+	<dd>Set left padding of 2 for the element</dd>
+	<dt><code>.mt-auto</code></dt>
+	<dd>Set auto margin-top for the element</dd>
+	<dt><code>.mb-sm-5</code></dt>
+	<dd>Set bottom margin of 5 for the element on small screens</dd>
+	<dt><code>.p-sm-3</code></dt>
+	<dd>Set padding of 3 for the element on small screens</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Appearance</h3>
+				<p class="p-0">
+					Example 1: No padding
+				</p>
```

Comment:
Apply the padding style along with a custom background color to show the spacing

```suggestion
				<p class="p-0 bg-info">
					Example 1: No padding
				</p>
```